packages/observable-client/src/chainHead/chainHead.ts

```diff
@@ -149,9 +153,20 @@ export const getChainHead$ = (chainHead: ChainHead) => {
     )
 
   const cache = new Map<string, Map<string, Observable<any>>>()
+
+  const stg = withRefcount(
+    withRecoveryFn(fromAbortControllerFn(lazyFollower("storage"))),
+  )
+  const getCodeHash = (blockHash: string): Observable<HexString> =>
+    // ":code" => "0x3a636f6465"
+    stg(blockHash, "hash", "0x3a636f6465", null).pipe(map((x) => x!))
```

Comment:
The `x!` made me think - is it possible for a substrate-like chain to not have a `:code` storage point, but use an alternative entry?

Maybe it's something we can revisit if we find such case, but we should probably start thinking about it.

Reply:
`:code` i one of the reserved storage entries that if it changes... it breaks everything: smoldot, clients, etc. So, I think we are golden.